fairworkflows/fairworkflow.py

```diff
@@ -473,7 +473,7 @@ def draw(self, filepath):
                 'Cannot produce visualization of RDF, you need to install '
                 'graphviz dependency https://graphviz.org/')
 
-    def publish_as_nanopub(self, use_test_server=False, **kwargs):
+    def publish_as_nanopub(self, use_test_server=False, force=False, **kwargs):
```

Comment:
I'm wondering if we could call this something that makes it clearer that steps are being published, like `publish_steps` (or even `force_publish_steps` although I think that makes it sound more dangerous than it is). With just `force` that sounds like it might publish the workflow regardless of whether steps are published - I'm not sure I'd necessarily expect that to mean it publishes the steps for me.

Reply:
You are right. I changed it to `pubish_steps`